fordel-domene/src/main/java/no/nav/foreldrepenger/mottak/domene/oppgavebehandling/OpprettGSakOppgaveTask.java

```diff
@@ -43,25 +42,23 @@
  * </p>
  */
 @Dependent
-@ProsessTask(TASKNAME)
+@ProsessTask(value = "integrasjon.gsak.opprettOppgave", maxFailedRuns = 2)
```

Comment:
Kanskje litt rart at denne kalles "value" og ikke "navn"?

Reply:
Easy living. Hvis du bytter fra value til navn så må alle annoteringer ha navn = ...

Reply:
Man kan vel ha begge ? Navn som alias for value?

Reply:
Ja, hadde vært fint om vi kunne hatt begge på en måte.

Reply:
Spring har @AliasFor("value") - men finner ikke tilsvarende utenfor Spring.
Annotation er gamle greier og "value" har spesialegenskaper. 
Jeg satt og lurte en del på om jeg skulle ha egenskaper i annotation eller default-metoder i ProsessTaskHandler-interface. Valgte annotation slik at man ikke trenger instansiere bønner - men kan gjøre getAnnotation på Class<? extends PTHandler>.